.clang-format

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
For easier review, please revert clang-format for now or create a clang-formatted base branch